test/integration/services/data/get-workload-reports-views.js

```diff
@@ -0,0 +1,60 @@
+const expect = require('chai').expect
```

Comment:
Yea, I kept meaning to change it. Its been like that since its incarnation, so will change in next commit.